corehq/ex-submodules/couchexport/shortcuts.py

```diff
@@ -30,6 +31,7 @@ def export_response(file, format, filename, checkpoint=None):
     if format.download:
         from corehq.util.files import safe_filename_header
         response['Content-Disposition'] = safe_filename_header(filename, format.extension)
+        response['Content-Length'] = os.fstat(file.fileno()).st_size
```

Comment:
It looks like `file` could be a stream, or some other file-like thing that doesn't necessarily have a `fileno()` method. Maybe see if you can get the size earlier, like near lines 25 and 29, depending on what it is.